tests/test_lifecycle.py

```diff
@@ -80,6 +87,41 @@ def test_create(self):
         # print(f"SEE V3 ON ZENODO: {res_json['links']['record_html']}")
         self.assertEqual(f"{data.version}-2", res_json["metadata"]["version"])
 
+    def test_metadata_fields(self):
```

Comment:
@nuest can you update the `test_create` to just include the more detailed metadata checks? This will reduce diff

Reply:
Like this?